astrix-gs/src/main/java/com/avanza/astrix/gs/GsBinder.java

```diff
@@ -117,7 +117,7 @@ public ServiceProperties createServiceProperties(String spaceUrl) {
 		private String versioned;
 		
 		public SpaceUrlBuilder(GigaSpace space) {
-			SpaceURL finderURL = space.getSpace().getFinderURL();
+			var finderURL = space.getSpace().getFinderURL();
```

Comment:
We can also remove the import of `SpaceURL` now.

Reply:
Good catch! Missing intelliJ setting. Fixed.